src/Symfony/Component/Routing/Alias.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Routing;
+
+use Symfony\Component\Routing\Exception\InvalidArgumentException;
+
+final class Alias
+{
+    /**
+     * @var string
+     */
+    private static $defaultDeprecationTemplate = 'The "%alias%" route alias is deprecated. You should stop using it, as it will be removed in the future.';
```

Comment:
This should be a constant.

Reply:
Agree. I just did the same as in DI's Alias class.

Reply:
Done.